src/Umbraco.Infrastructure/Services/MemberEditingService.cs

```diff
@@ -393,4 +399,6 @@ private async Task<bool> ChangePassword(IMember member, string? oldPassword, str
 
         return true;
     }
+
+    private static bool IsEmailValid(string email) => new EmailAddressAttribute().IsValid(email);
```

Comment:
Is this method needed needed anymore? We can use the new extension method like `if (model.Email.IsEmail() is false)`

Reply:
Nope it's not needed, good catch! 💪 
Removed 🚀